docs/performance.md

```
@@ -0,0 +1,437 @@
+
+[Go back to navigation](#GraphQL)
+
+### GraphQL Create Customer Test
+
+| Test type | Target RPS | Real RPS | Virtual Users | Rise | Plateau | Fall | P(99) |
+| --------- | ---------- | -------- | ------------ | ---- | ------- | ---- | ----- |
+| smoke     | 5          | 5.0      | 3            | -    | 10s     | -    | 101ms |
+| average   | 15         | 12.4     | 15           | 2s   | 8s      | 2s   | 60ms  |
+| stress    | 50         | 40.6     | 50           | 3s   | 10s     | 3s   | 82ms  |
+| spike     | 100        | 49.8     | 100          | 3s   | -       | 3s   | 223ms |
+[Go back to navigation](#GraphQL)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid
Expected: #graphql; Actual: #GraphQL

(MD051, link-fragments)